app/viewers/embed/viewer/file.rb

```diff
@@ -36,6 +36,17 @@ def display_file_search?
                                  @purl_object.contents.map(&:files).flatten.length >= min_files_to_search
       end
 
+      def message
+        return { type: 'embargo', message: embargo_message } if @purl_object.embargoed?
+
+        if @purl_object.location_restriction
```

Comment:
Won't this show the banner even when the person is in this access location?  Shouldn't we run the auth/probe service on stacks to see if they are in a restricted situation or not?

Reply:
Nope. The ticket says ```If possible the banner should persist even when the user is in the authorized location, as there should be a persistent indicator that the items are restricted.```